browse/templates/abs/extra_services.html

```diff
@@ -189,11 +185,14 @@ <h2>Access Paper:</h2>
       <div class="abs-license">
         {%- if withdrawn -%}
         <div hidden>No license for this version due to withdrawn</div>
-        {%- elif abs_meta.license.icon_uri_path -%}
```

Comment:
Please don't change the license display logic unless there is a Jira ticket where a bug is being fixed.
The requested code change is to reinstate the removed lines on 192, 193 and 195 and to remove the changes in the new version on 189 to 193.